app/models/miq_alert.rb

```diff
@@ -127,10 +129,16 @@ def self.assigned_to_target(target, event = nil)
       alert_ids = profiles.flat_map { |p| p.members.pluck(:id) }.uniq
 
       if alert_ids.empty?
+        _log.warn("#{log_header} #{log_target} No alerts are assigned to target through alert profiles!")
         none
       else
+        msg = "Filtering for enabled alerts"
         scope = where(:id => alert_ids, :enabled => true, :db => target.class.base_model.name)
-        scope = scope.where("responds_to_events like ?", "%#{event}%") if event
+        if event
+          scope = scope.where("responds_to_events like ?", "%#{event}%")
```

Comment:
Just out of curiousity what does `responds_to_events` typically look like?  From the specs I see `"xxx|vm_perf_complete|zzz"` and `_hourly_timer_`

Reply:
Usually underscored values, but a few aren't.  Look at the `responds_to_events` values here:

https://github.com/ManageIQ/manageiq/blob/e6e94e9388055ce5ab3bc13281333c8c311cc301/app/models/miq_alert.rb#L409-L483

I don't know if there are more.